base/Dockerfile.v2

```diff
@@ -16,6 +16,8 @@ LABEL org.opencontainers.image.vendor="Netdata Inc."
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN echo 'deb http://deb.debian.org/debian bookworm-backports main' > /etc/apt/sources.list.d/backports.list
```

Comment:
This should be part of the same `RUN` command as the next one, because it should not be a separate layer.

Also, if possible, we should be generating the suite name (`bookworm-backports`) so that it gets updated correctly when we bump the base image version.